docs/package.json

```diff
@@ -10,12 +10,13 @@
     "files": [],
     "scripts": {
         "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
-        "watch:sass": "sass --watch --poll src/assets/scss:src/assets/css",
+        "watch:sass": "sass --watch --poll src/assets/scss:_site/assets/css",
         "watch:eleventy": "eleventy --serve --port=2023",
-        "build:sass": "sass --style=compressed src/assets/scss:src/assets/css --no-source-map",
```

Comment:
Maybe keep it at `src/assets/css` since we copy these assets to `_site`?

https://github.com/eslint/eslint/blob/e94a4a95ee301b0344d3292c37a0b29d8e18ab30/docs/.eleventy.js#L342

Reply:
Yes, if you look in the build folder, the scss files are always copied. I tried looking at a way to fix it before, but I'm not familiar enough with eleventy

Reply:
Went back and adjusted it so it chains through for the built CSS, but the SCSS is no longer copied through